source/languages/en/riak/dev/search/search-data-types.md

```diff
@@ -965,7 +965,7 @@ This example will build on the example in the [[Using Data Types]]
 tutorial. That tutorial walks you through storing CMS-style user data in
 Riak [[maps|Using Data Types#Maps]], and we'd suggest that you
 familiarize yourself with that tutorial first. More specifically, user
-data is stored in the following fields in each users's map:
+data is stored in the following fields in each users' map:
```

Comment:
Originally, this should have been written user's given the context. Would you mind updating that, please?